src/Service.Host/client/src/components/POReqSearch.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState, Fragment, useEffect } from 'react';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import { makeStyles } from '@mui/styles';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import Link from '@mui/material/Link';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+    Page,
+    InputField,
+    Loading,
+    utilities,
+    CreateButton,
+    collectionSelectorHelpers
+} from '@linn-it/linn-form-components-library';
+import Divider from '@mui/material/Divider';
+import { Link as RouterLink } from 'react-router-dom';
+import purchaseOrderReqsActions from '../actions/purchaseOrderReqsActions';
+import history from '../history';
+import config from '../config';
+
+function POReqSearch() {
+    const useStyles = makeStyles(theme => ({
+        button: {
+            marginLeft: theme.spacing(1),
+            marginTop: theme.spacing(4)
+        },
+        a: {
+            textDecoration: 'none'
+        }
+    }));
+
+    const classes = useStyles();
+
+    const loading = useSelector(state =>
+        collectionSelectorHelpers.getSearchLoading(state.purchaseOrderReqs)
+    );
+
+    const results = useSelector(state =>
+        collectionSelectorHelpers.getSearchItems(state.purchaseOrderReqs)
+    );
+    const applicationState = useSelector(state =>
+        collectionSelectorHelpers.getApplicationState(state.purchaseOrderReqs)
+    );
+
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(purchaseOrderReqsActions.fetchState());
+    }, [dispatch]);
+
+    const [options, setOptions] = useState({ reqNumber: '', part: '', supplier: '' });
+
+    const handleOptionsChange = (propertyName, newValue) =>
+        setOptions({ ...options, [propertyName]: newValue });
+
+    const createUrl = utilities.getHref(applicationState, 'create');
+
+    useEffect(() => {
+        if (
+            options.reqNumber.length > 2 ||
+            options.part.length > 2 ||
+            options.supplier.length > 2
```

Comment:
I wanted to use useSearch for this, but couldn't get it working. This works but if folk do want to search by more than one field it's gonna fire off a request for every character they type in the second field. Once I've spoken to folk the search fields might get changed anyway though so I can come back to it